srv/salt/ceph/time/README

```diff
@@ -1,12 +1,24 @@
-Set up NTP to manage system time
+Set up Chrony or ntp to manage system time.  Chrony is preferred.
 
-Ceph relies on loosely synchronized clocks. DeepSea will by default start an
-NTP client on each cluster node and point it to the NTP server running on the
-Salt Master if and only if the ntpd.service systemd unit is not running.
+Ceph relies on synchronized clocks. DeepSea will by default start a chrony 
+client on each minion.  If chrony is not available, then ntp must be selected
+in the pillar.  In either case, the default time server is the Salt master 
+unless overridden.  The Salt master must be configured manually.  (Salt can be
```

Comment:
Should we add quick instructions on how to setup a chrony server? Eric suggests either adding something in the wiki, or a prewritten configuration file under `contrib/`.

Reply:
We should also add the info that the default setup that deepsea creates is creating a single point of failure (admin down = no timesync) and due to that must not be used that way in production.

Reply:
I am fine with adding sample configuration files for time servers under contrib and putting a README there or adding something to the wiki to reference it.  

The default setup is just that.  It's the default which is suitable for a Proof of Concept.  I expect admins have a time server setup already and that the admin node would be a client as well.

Reply:
@swiftgist's suggestion sounds good to me.